bindata/etcd/cluster-backup.sh

```diff
@@ -34,10 +34,9 @@ function backup_latest_kube_static_resources {
 
   LATEST_RESOURCE_DIRS=()
   for RESOURCE in "${RESOURCES[@]}"; do
-    # shellcheck disable=SC2012
-    LATEST_RESOURCE=$(ls -trd "${CONFIG_FILE_DIR}"/static-pod-resources/"${RESOURCE}"-[0-9]* | tail -1) || true
+    LATEST_RESOURCE=$(crictl inspect "$(crictl ps --name "${RESOURCE}"$ -q)" | jq -r '.status.mounts | .[] | select (.containerPath=="/etc/kubernetes/static-pod-resources") | .hostPath') || true
     if [ -z "$LATEST_RESOURCE" ]; then
-      echo "error finding static-pod-resource ${RESOURCE}"
+      echo "error finding static-pod-resource for the ${RESOURCE} pod"
```

Comment:
small nit maybe we should add "please check if it is running" here? That seems to be the primary failure for crictl to error because the container is maybe crash looping thus not running.